### PR TITLE
[API][SIMS #1222] Federal Field of Study Code Part 2

### PR DESCRIPTION
**Pull request by henryoforeh-aot**
- Replaced CIP Code with Field of Study Code in the query that is part of the process for generating full time e-cert file.

### CODE REVIEW

File: sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts [hunk 1]
```diff
@@ -16,7 +16,6 @@ import {
   ECERT_FULL_TIME_FEEDBACK_FILE_CODE,
   ECERT_PART_TIME_FEEDBACK_FILE_CODE,
   getISODateOnlyString,
-  getFieldOfStudyFromCIPCode,
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts:19]: Can this method be removed also?

Comment (henryoforeh-aot) [reply sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts:19]: Yes, I forgot. I will remove it.

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by andrewsignori-aot — APPROVED** [review]
Nice work, looks good 👍

File: sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts [hunk 2]
```diff
@@ -235,7 +232,7 @@ export class ECertFileHandler extends ESDCFileHandler {
       educationalEndDate: offering.studyEndDate,
       federalInstitutionCode: offering.institutionLocation.institutionCode,
       weeksOfStudy: application.currentAssessment.assessmentData.weeks,
-      fieldOfStudy,
+      fieldOfStudy: offering.educationProgram.fieldOfStudyCode,
```
Comment (dheepak-aot) [inline sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts:235]: fieldOfStudyCode is not added to the select which is used by this method.

image

**Review by dheepak-aot — APPROVED** [review]
LGTM @henryoforeh-aot 👍

**Review by ann-aot — APPROVED** [review]
LGTM 👍

**Pull request merged**